### DISCUSSION TITLE
Allow specifying a Proxy Server

### DISCUSSION THREAD

**Discussion by martintoelk** [microsoft/powerplatform-vscode#550] [Ideas]
Hi PAC CLI Team,

most big coorporations requires usages of a proxy server. At the moment you have to manually touch the app.config located at:
`C:\Users\{USERNAME}\.nuget\packages\microsoft.powerapps.cli\{PAC CLI Version}\tools\pac.exe.config`

and add the following xml tag

`	<system.net>
		<defaultProxy enabled="true" useDefaultCredentials="true">
		</defaultProxy>
	</system.net>`

This step has to be repeated for each new version. So it would be great, if we can directly configure the proxy via a pac cli statement
for example:
pac config proxy add -useDefaultCredentials "true"

BR

Martin

**Comment by TomProkop** [comment]
For others who use VS Code sourced PAC CLI this is the path:

C:\Users\{USERNAME}\AppData\Roaming\Code\User\globalStorage\microsoft-isvexptools.powerplatform-vscode\pac\tools

If you want to run pac tools like cmt (Configuration Migration Tool) you need to do the same in the respective config files. Keep in mind that with new versions you will need to do it again.

C:\Users\{USERNAME}\AppData\Local\Microsoft\PowerPlatform\CMT\9.1.0.110\tools\DataMigrationUtility.exe.config
